base_delivery_carrier_label/models/stock_move_line.py

```diff
@@ -2,47 +2,32 @@
 # Copyright 2016 Camptocamp SA
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
-import logging
 
 from odoo import fields, models
-
-_logger = logging.getLogger(__name__)
+from odoo.tools import float_round
 
 
 class StockMoveLine(models.Model):
     _inherit = "stock.move.line"
 
-    weight = fields.Float(digits="Stock Weight", help="Weight of the pack_operation")
+    weight = fields.Float(digits="Stock Weight", help="Weight in kg")
```

Comment:
Maybe I'm wrong, but it can be kg _or_ pound. Don't you think ?

Odoo support both main Unit of Weight.

You can display the UoM name similar as in product.template : https://github.com/odoo/odoo/blob/14.0/addons/product/models/product_template.py#L95

And see : https://github.com/odoo/odoo/blob/14.0/addons/product/models/product_template.py#L293-L303

regards.